release_notes_clear.py

```diff
@@ -107,7 +107,7 @@ def remove_releaseNotes_folder(folder_path, files_extension):
 
 
 def main(root_dir):
-    yml_folders_to_scan = ['Integrations', 'Playbooks', 'Scripts', ] # yml
+    yml_folders_to_scan = ['Integrations', 'Playbooks', 'Scripts', 'TestPlaybooks'] # yml
```

Comment:
since when do we have releasenotes on test playbook?

Reply:
we don't - but devs put it anyway